src/metabase/automagic_dashboards/names.clj

```diff
@@ -0,0 +1,164 @@
+(ns metabase.automagic-dashboards.names
+  (:require
+   [clojure.string :as str]
+   [java-time.api :as t]
+   [metabase.automagic-dashboards.util :as magic.util]
+   [metabase.mbql.util :as mbql.u]
+   [metabase.query-processor.util :as qp.util]
+   [metabase.util.date-2 :as u.date]
+   [metabase.util.i18n :refer [deferred-tru tru]]
+   [toucan2.core :as t2]))
+
```

Comment:
This namespace is just all of the string and naming functions pulled out of automagic.core to reduce clutter.